src/merida/metadata_cls.py

```diff
@@ -106,8 +106,6 @@ def __init__(self, internal_lightcurve_name, *,
         self.alert_id2 = self.metadata['alert_id2'].values[0]
         self.alert_x2 = self.metadata['alert_x2'].values[0]
         self.alert_y2 = self.metadata['alert_y2'].values[0]
-        self.ROW_IDX = self.metadata['ROW_IDX'].values[0]
```

Comment:
These seemed to be metadata rows that don't exist in the IPAC version and just seemed to increment with the light curves. I'm guessing these were an artifact that was adding during the original metadata file creation? I don't believe they are needed.